package.json

```diff
@@ -18,7 +18,8 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "cdk": "cdk",
-    "generate-schema": "ts-node ./src/scripts/generate-json-schema.ts"
+    "schemas": "ts-node ./src/scripts/generate-json-schema.ts",
```

Comment:
Oh I quite liked the fact that the word `generate` was in here! What's the reason to change it (I don't feel super strongly, just interested).

Reply:
I've renamed back to `generate-schema`